### PR TITLE
Add Support for Anko Layouts

### LINKED ISSUE #1320: Anko Layouts
**Describe the feature you'd like**
Use Anko DSL Layouts instead of xml especially for recyclerview items 
Why Anko Layouts ?
https://android.jlelse.eu/400-faster-layouts-with-anko-da17f32c45dd
**Would you like to work on the issue?**
- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by aggarwalpulkit596**
Fixes #1320 

Changes: Converted Item Card Event to Anko DSL

Screenshots for the change:

### CODE REVIEW

**Comment by aggarwalpulkit596** [comment]
@iamareebjamal review this you might have to run this on an actual device to see the difference in results and also how can i test it with actual result currently i was able to test with sample data only

File: app/src/main/java/org/fossasia/openevent/general/event/ItemCardEventsUi.kt [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+import android.annotation.SuppressLint
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.constraintlayout.widget.ConstraintLayout
+import org.fossasia.openevent.general.R
+import org.jetbrains.anko.*
```
Comment (ShridharGoel) [inline app/src/main/java/org/fossasia/openevent/general/event/ItemCardEventsUi.kt]: Build is failing because of wildcard import

**Comment by aggarwalpulkit596** [comment]
Thanks for pointing it out but i already know that and i am waiting for it review first and PR is incomplete as of now

**Comment by aggarwalpulkit596** [comment]
@iamareebjamal please review this

**Comment by iamareebjamal** [comment]
Yeah I don't think it looks good or maintainable. Plus, I'm planning on using data binding later which we can't do here

**Comment by aggarwalpulkit596** [comment]
We can use databinding as well checkout this video https://youtu.be/dYskPdOSvo4

**Comment by iamareebjamal** [comment]
You can see the example in the video itself how verbose and unpleasant it is. People won't want to use databinding if they had to use it like that. Also, you can't mix anko layouts with normal layouts AFAIK. I want to include a layout and bind parameters in the XML, I can't do that

**Comment by aggarwalpulkit596** [comment]
We can use include as well but nm I'll close this PR

**Comment by iamareebjamal** [comment]
I vote against anko layouts because where anko layouts shine is when you have heavy dynamic layouts or want to remove overhead of layout inflation in case of simple layouts, etc. We don't have dynamic layouts, nor is layout inflation a bottleneck for us

**Comment by iamareebjamal** [comment]
You can add anko layouts in new layouts we add for simple layouts but I don't think we should convert existing layouts

**Pull request closed without merging**